packages/design/src/DataTable/DataTable.story.tsx

```diff
@@ -42,6 +42,65 @@ export const DataTable = () => {
   );
 };
 
+export const DataTableWrappedLabels = () => {
```

Comment:
Nit: This isn't really a "new" feature that needs to be documented separately in our storybook. I would update `DataTable` to have a long tag value.